projects/ng-polymorpheus/tsconfig.lib.json

```diff
@@ -15,7 +15,8 @@
         "fullTemplateTypeCheck": true,
         "strictInjectionParameters": true,
         "enableResourceInlining": true,
-        "enableIvy": false
+        "enableIvy": true,
+        "compilationMode": "partial"
```

Comment:
What is `partial`? Is there `full-100%-ivy mode`?

Reply:
No. That is how you are supposed to compile Ivy libraries that go to npm:
https://blog.lacolaco.net/2021/02/angular-ivy-library-compilation-design-in-depth-en/